app/controllers/api/v1/concerns/study_aware.rb

```diff
@@ -5,7 +5,8 @@ module StudyAware
         extend ActiveSupport::Concern
 
         def set_study
-          @study = Study.find_by(accession: params[:study_id])
+          # allow either accession or id
+          @study = Study.find_by(accession: params[:study_id]) || Study.find_by(id: params[:study_id])
```

Comment:
slightly easier way to do this: 
```suggestion
          @study = Study.any_of({accession: params[:study_id]},{id: params[:study_id]}).first
```